src/components/RightPane/SectionTable/PrereqTree.css

```diff
@@ -0,0 +1,194 @@
+:root {
+    --grey-light-theme: #8d8d8d;
+    --background-grey-light-theme: #e0e0e0;
+  }
+
+.dependency-list-branch {
+    margin: auto;
+}
+
+.prereq {
+    display: flex;
+    flex-direction: column;
+    overflow-x: auto;
+}
+
+.prereq-clump {
+    display: flex;
+    position: relative;
+    margin-left: 20px;
+}
+
+.prereq-clump::before {
+    content: '';
+    position: absolute;
+    background: var(--grey-light-theme);
+    top: 50%;
+    left: -1.5rem;
+    width: 0.5rem;
+    height: 1px;
+}
+
+.prereq-list {
+    margin: 0;
+    padding: 0;
+}
+
+.prereq-branch,
+.dependency-branch {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0.125rem 0.25rem;
+    margin: 0 0.25rem 1px 0.25rem;
+    border-radius: 0.25rem;
+    flex: 0 0 auto;
+    border: 0;
+}
+
+.dependency-needs::before {
```

Comment:
I don't like pseudoselectors because of their complexity but if there isn't any more elegant of a way to implement whatever style you're trying to pull off here then it's fine. Just from scanning this casually is this supposed to be the lines connecting the nodes in the tree? If so add a comment because it isn't very obvious what these are supposed to be for.